core/src/main/scala/org/genivi/sota/core/CampaignResource.scala

```diff
@@ -82,6 +93,18 @@ class CampaignResource(namespaceExtractor: Directive1[AuthedNamespaceScope],
     complete(CampaignStats.get(id, updateService))
   }
 
+  def toggleDelta(id: Campaign.Id): Route = {
+    parameters(('deltaFromName.as[String Refined PackageId.ValidName].?,
```

Comment:
btw `String Refined PackageId.ValidName` is the same as `PackageId.Name` similarly
`String Refined PackageId.ValidVersion` is the same as `PackageId.Version`